MetaMask/metamask-mobile
# Support sepolia network

**Description**
Add support for sepolia network
This PR have merged in it this [PR](https://github.com/MetaMask/metamask-mobile/pull/6123)

**Proposed Solution**
Add support for sepolia network and remove code with rinkeby, ropsten and kovan
Also removed deprecated warning when the user was on the deprecated test networks.
Added and changed test cases with Sepolia Network.

**Screenshots/Recordings**
(**Update Sepolia Icon on the last commit**)
<img width="250" alt="image" src="https://user-images.githubusercontent.com/46944231/231230837-3816c77e-7666-4967-8c4d-26ce99ae7fad.png">

**Test Cases**
Case1:
 * Change to Sepolia Network 
 * Tokens are fetched correctly
 * Go to send flow
 * Expected to be able to execute a transaction

Case2:
 * Change to Ethereum Mainnet 
 * Tokens are fetched correctly
 * Go to send flow
 * Expected to be able to execute a transaction

Case3:
 * Change to Goerli Network 
 * Tokens are fetched correctly
 * Go to send flow
 * Expected to be able to execute a transaction


**Recordings**
* Transactions with sepolia network: https://recordit.co/MGvUDm1hWE
* Transaction with mainnet: https://recordit.co/wiDWJJZEgr
* Transaction with goerli network: https://recordit.co/sHLkBJhcpt

Case4:
Importing ERC20 token on Sepolia and sending transaction - https://recordit.co/2gNSkJ4cRY and https://recordit.co/6jF4tZckR2

Case5:
connect to the test dapp and perform dapp interactions. 
Interactions include: Creating token, approving tokens and contracts (1155's, Multisig, collectibles) performing a multisig
transaction - https://recordit.co/DIq4Df9B7w

Case 6:
user should not have the ability to edit or delete the sepolia network - https://recordit.co/iMuUdMA9V6

Case 7:
the API keys for sepolia should not be exposed in the RPC URL endpoint. 
<img width=250 src="https://user-images.githubusercontent.com/46944231/230802603-caef7384-65b5-4a20-b633-9305971a1df9.png" />

Case 8:
Rinkeby, Ropsten and Koran should no longer be in on the networks page: Settings > networks. 
<img width=250 src="https://user-images.githubusercontent.com/46944231/231230837-3816c77e-7666-4967-8c4d-26ce99ae7fad.png" />


**Issue**

Progresses #5406 

**Checklist**
